nntrainer/layers/fc_layer.cpp

```diff
@@ -133,8 +179,12 @@ void FullyConnectedLayer::forwarding(RunLayerContext &context, bool training) {
       context.getWeightObject(weight_idx[FCParams::weight]).getOutputAxis();
 
     weight.dequantize(weight_, axis);
+    if (!std::get<props::LoraRank>(fc_props).empty())
+      forwarding_lora(context, weight_);
```

Comment:
This is not the right way to implement LoRA. Overall, this is right but computing Lora's weight every time is not necessary. We only need this in finalize process of FC once. There is another way to implement it and I think it might be better to talk about it more.

Reply:
Could you explain more details about _'separate training and inference'_? In `edc76c`, I revised the code in the way of separately forward baseline weight (W) and lora parameters (A@B). Still, I think there is more points that I have to consider to update the LoRA in terms of the _mode_. Thanks!